opensearch/src/main/java/org/opensearch/sql/opensearch/setting/OpenSearchSettings.java

```diff
@@ -123,6 +146,10 @@ public OpenSearchSettings(ClusterSettings clusterSettings) {
         METRICS_ROLLING_WINDOW_SETTING, new Updater(Key.METRICS_ROLLING_WINDOW));
     register(settingBuilder, clusterSettings, Key.METRICS_ROLLING_INTERVAL,
         METRICS_ROLLING_INTERVAL_SETTING, new Updater(Key.METRICS_ROLLING_INTERVAL));
+    register(settingBuilder, clusterSettings, Key.DATASOURCES_URI_ALLOWHOSTS,
+        DATASOURCE_URI_ALLOW_HOSTS, new Updater(Key.DATASOURCES_URI_ALLOWHOSTS));
+    settingBuilder.put(Key.CLUSTER_NAME, ClusterName.CLUSTER_NAME_SETTING);
```

Comment:
why SQL plugin handle cluster_name setting?

Reply:
SQL Plugin requires this in StorageEngineFactory class in datasource connector modules.